src/porepy/geometry/map_geometry.py

```diff
@@ -260,18 +275,26 @@ def project_line_matrix(
     reference: Optional[np.ndarray[Any, np.dtype[np.float64]]] = None,
 ) -> np.ndarray[Any, np.dtype[np.float64]]:
     """Project the points on a line using local coordinates.
-
     The projected points are computed by a dot product.
-    example: np.dot( R, pts )
-
+    
+    FIXME: does the example below presents the correct usage?
```

Comment:
I will have a look at this in a separate stage; I'll leave it now so that I do not forget.